politeiad/v2.go

```diff
@@ -506,9 +506,18 @@ func (p *politeia) handlePluginWrite(w http.ResponseWriter, r *http.Request) {
 	payload, err := p.backendv2.PluginWrite(token, pw.Cmd.ID,
 		pw.Cmd.Command, pw.Cmd.Payload)
 	if err != nil {
-		respondWithErrorV2(w, r,
-			"handlePluginWrite: PluginWrite: %v", err)
-		return
+		switch {
```

Comment:
Why do you need this?  Doesn't the `respondWithErrorV2()` take care of this?

Reply:
Did you confirm that `respondWithErrorV2()` handles this correctly?  I think it does, but I could be wrong.

Reply:
Yes you are right mate. Just needed to add the new error code to `convertErrorToV2`. My bad. Testing from scratch will report back in a few

Reply:
All good 👍